File: system/framework/shops/atb/AtbItemModel.php
<?php

namespace framework\shops\atb;

class AtbItemModel{
    public ?int $inshopid;
    public ?string $label;
    public ?string $image;
    public ?int $shopcategoryid;
    public ?string $brand;
    public ?string $country;
    public ?string $url;

    public function __construct(
        ?int $inshopid = null,
        ?string $label = null,
        ?string $image = null,
        ?int $shopcategoryid = null,
        ?string $brand = null,
        ?string $country = null,
        ?string $url
    ) {
        $this->inshopid = $inshopid;
        $this->label = $label;
        $this->image = $image;
        $this->shopcategoryid = $shopcategoryid;
        $this->brand = $brand;
        $this->country = $country;
        $this->url = $url;
    }
}

File: priceapp.proxy.Models/AtbItemModel.cs
namespace priceapp.proxy.Models;

public class AtbItemModel
{
    public int Id { get; set; }
    public int InternalId { get; set; }
    public string Label { get; set; }
    public string Image { get; set; }
    public int Category { get; set; }
    public string? Brand { get; set; }
    public string? Country { get; set; }
}